src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -116,18 +118,17 @@
     in your code or check the credentials into your source control. As an alternative, consider using
     [managed identities](https://aka.ms/azadsp-managed-identities) if available to avoid the need to use credentials.
 examples:
-  - name: Append a certificate to the application with the certificate string.
-    text: az ad app credential reset --id 00000000-0000-0000-0000-000000000000 --cert "MIICoT..." --append
-  - name: Append a certificate to the application with the certificate file.
-    text: |-
-        az ad app credential reset --id 00000000-0000-0000-0000-000000000000 --cert "@~/cert.pem" --append
-        `cert.pem` contains the following content
-        -----BEGIN CERTIFICATE-----  <<< this line is optional
-        MIICoT...
-        -----END CERTIFICATE-----    <<< this line is optional
-  - name: append or overwrite an application's password or certificate credentials (autogenerated)
-    text: az ad app credential reset --id 00000000-0000-0000-0000-000000000000
-    crafted: true
```

Comment:
This auto-generated help is confusing. Unclear about whether it **appends or overwrites**, whether it operates on **password or certificate credentials**.